paper-drawer-panel.html

```diff
@@ -437,7 +437,6 @@
           this.selected = this.defaultSelected;
         }
 
-        this.setScrollDirection(this._swipeAllowed() ? 'y' : 'all');
```

Comment:
This function is defined in gestures: https://github.com/Polymer/polymer/blob/master/src/standard/gestures.html#L523

It sets `touch-action` property to either `pan-y` in the "swipe allowed" case, or `touch-action: auto` in the other case.